agent/connect/ca/testing.go

```diff
@@ -162,7 +163,7 @@ func runTestVault(t testing.T) (*TestVaultServer, error) {
 	}
 	t.Cleanup(func() {
 		if err := testVault.Stop(); err != nil {
-			t.Log("failed to stop vault server: %w", err)
+			t.Logf("failed to stop vault server: %v", err)
```

Comment:
not sure why `%w` was not working here, but it was not formatting correctly, so `%v` should be fine. Maybe `%w` only works in `fmt.Errorf` ?